### PR TITLE
Update Web and App SDK docs to reflect current retired/retiring state

### PR DESCRIPTION
**Pull request by nitmeht**
AppSDK has been retired. WebSDK is on schedule to retire on September 30, 2024. Hence, we are updating the documentation to reflect the status.

### CODE REVIEW

File: Skype/Trusted-Application-API/docs/AnonymousMeetingJoin.md
Comment (rasinive) [inline Skype/Trusted-Application-API/docs/AnonymousMeetingJoin.md:1]: Remove Skype Web SDK, App SDK from here

File: Skype/WebSDK/docs/ReleaseNotes.md
Comment (rasinive) [inline Skype/WebSDK/docs/ReleaseNotes.md:1]: Why did you leave this file?

**Review by rasinive — APPROVED** [review]

File: Skype/WebSDK/docs/ReleaseNotes.md
Comment (nitmeht) [reply Skype/WebSDK/docs/ReleaseNotes.md:1]: I left it thinking that the SDK is not yet decommissioned. Should I remove it at this time?

File: Skype/Trusted-Application-API/docs/AnonymousMeetingJoin.md
Comment (nitmeht) [reply Skype/Trusted-Application-API/docs/AnonymousMeetingJoin.md:1]: done

File: Skype/WebSDK/docs/ReleaseNotes.md
Comment (rasinive) [reply Skype/WebSDK/docs/ReleaseNotes.md:1]: I would remove so we don't do any updates later to it anymore. 
Just make sure we have a copy of the files and we can provide the info if requested somehow.

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**